src/express/routes/articles.js

```diff
@@ -204,6 +205,9 @@ articlesRouter.post(
 
         await api.postComment({userId, articleId, text});
 
+        const [freshComments, hotArticles] = await Promise.all([api.getFreshComments(), api.getMostDiscussed()]);
+        io.emit(EventNames.COMMENT_ADD, JSON.stringify({freshComments, hotArticles}));
```

Comment:
"There is no need to run JSON.stringify() on objects as it will be done for you."